spec/controllers/email_authentications/invitations_controller_spec.rb

```diff
@@ -102,22 +103,22 @@
       end
 
       it 'responds with bad request email already exists' do
-        EmailAuthentication.create!(email: email, password: Faker::Internet.password)
+        EmailAuthentication.create!(email: email, password: generate(:strong_password))
```

Comment:
Curious, where does `generate` come from? Does `strong_password` register some sort of generator with Rspec or FactoryBot?